etcdmgmt/etcd-mgmt.go

```diff
@@ -28,6 +64,19 @@ func StartEtcd() error {
 
 	initialAdvPeerUrls := "http://" + hostname + ":2380"
 
+	// Checking health of etcd cluster. If etcd daemon already running then
```

Comment:
I'd already given you my opinion on this. Attempting to store information about an etcd instance in etcd is not right. Just to make it clear to you, when you need to decide if something must be stored in etcd, ask the question 'Is this information relevant to the whole cluster or just to this one peer?'. Only store information relevant to the cluster in etcd. Everything else, use local storage, possibly using `/var/lib` or `/var/run`.

I'd also told about my preferred approach to check etcd liveness on GD2 startup. In simple terms,
- Check if you can connect to etcd and check for cluster health (this is the same as what you are doing to check if etcd is operational after you start it)
- Start etcd
- Wait for etcd to become operational